mujinvisioncontrollerclient/visioncontrollerclient.py

```diff
@@ -207,19 +241,26 @@ def StartObjectDetectionTask(self, taskId=None, systemState=None, visionTaskPara
         return self._ExecuteCommand(command, timeout=timeout)
 
     def StartContainerDetectionTask(self, taskId=None, systemState=None, visionTaskParameters=None, timeout=2.0, **ignoredArgs):
-        # type: (Optional[types.SystemState], Optional[types.visionTaskContainerDetectionParametersSchema], float, Dict) -> Optional[Dict]
+        # type: (Optional[str], Optional[Union[types.SystemState, Dict]], Optional[Union[types.visionTaskContainerDetectionParametersSchema, Dict]], float, Any) -> Optional[Dict[str, str]]
```

Comment:
Why Optional[Union[types.SystemState, Dict]]? I think the docstring was outdated here.

Reply:
Oh, I see. Fixed then.